docker/docker/src/main/java/org/arquillian/cube/docker/impl/docker/DockerClientExecutor.java

```diff
@@ -465,8 +465,8 @@ public void stopContainer(String containerId) {
         this.dockerClient.stopContainerCmd(containerId).exec();
     }
 
-    public void removeContainer(String containerId) {
-        this.dockerClient.removeContainerCmd(containerId).exec();
+    public void removeContainer(String containerId, boolean removeVolumes) {
```

Comment:
One thing that I have thought today. Maybe it would be better to pass CubeContainer object like it happens with create. In this way if we ever need to do any other tasks when removing the container, we don't need to expand the method. Maybe for now we can leave as is and in the future if this happens refactor with CubeContainer class. @aslakknutsen what do you think?